e2e/client/ippool.go

```diff
@@ -13,9 +13,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func isIPPoolAllocationsEmpty(k8sIPAM *kubeClient.KubernetesIPAM, ipPoolName string) wait.ConditionFunc {
+func isIPPoolAllocationsEmpty(k8sIPAM *kubeClient.KubernetesIPAM, ipPoolCIDR string) wait.ConditionFunc {
```

Comment:
You'll need to parameterize this function with the network name, since I expect you to add a new e2e test where the same range can be used by different networks.

I think you should add a new type (struct) featuring both the CIDR and the name. Then pass this around.

I think its idiomatic, and fits well with the chosen model.

Reply:
✅